documentation/src/main/asciidoc/topics/ref_query_ickle_syntax.adoc

```diff
@@ -128,6 +128,17 @@ in the same `Object[]` returned hit.
 SELECT b, version(b) FROM org.infinispan.sample.Book b WHERE b.title like '%Data Grid%'
 ----
 
+=== Project the score
+If the query is indexed, it is possible to project the score obtained by each matching together with other projections.
+It can be used for instance to project the cache entry value together with the score
+in the same `Object[]` returned hit.
+
+[source,sql]
+----
+# return the cache entry value and the cache entry version
+SELECT b, score(b) FROM org.infinispan.sample.Book b WHERE b.title like '%Data Grid%'
+----
+
```

Comment:
this example doesn't seems correct for vector search and score projection

Reply:
We're not in the vector search chapter, so I think that the example should be generic for any kind of query.
By the way the comment is wrong, So I'm fixing it :P

Reply:
I'm adding also a paragraph to the vector chapter to show an example of the combination of the use of the score projection with the vector search query.